python/lsst/afw/table/key.py

```diff
@@ -0,0 +1,42 @@
+from __future__ import absolute_import
+
+from .tableLib import *
+from ._key import *
+
+from .schema import *
+
+def key_eq(self, other):
+    """
+    Compare two keys. If the key types are the same they are checked for equality using the
+    C++ layer. Otherwise NotImplemented is returned.
```

Comment:
Fixed the comment. This is directly from the SWIG code and there is a test for this explicitly, so I recommend leaving it for now.